### PR TITLE
add list of supported eth tokens

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Deploy ERC20 to ropsten.
Add them to list.

### CODE REVIEW

File: deploy/db/sql/init/populate.sql [hunk 1]
```diff
@@ -6,5 +6,14 @@ INSERT INTO notary.wallets (wallet, irohausername)
 VALUES ('0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e', 'user2@notary');
 
 -- Populate tokens table
-INSERT INTO notary.tokens (wallet, token)
-VALUES ('0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e', 'tkn');
+INSERT INTO notary.tokens (wallet, token) VALUES
```
Comment (luckychess) [inline deploy/db/sql/init/populate.sql]: What about lines above, do we need them?

File: src/main/kotlin/sidechain/eth/util/DeployHelper.kt [hunk 2]
```diff
@@ -65,16 +65,16 @@ class DeployHelper(ethereumConfig: EthereumConfig, ethereumPasswords: EthereumPa
...
      */
-    fun deployBasicCoinSmartContract(): BasicCoin {
+    fun deployERC20TokenSmartContract(): BasicCoin {
         return contract.BasicCoin.deploy(
             web3,
             credentials,
             gasPrice,
             gasLimit,
-            BigInteger.valueOf(1000),
+            BigInteger.valueOf(Long.MAX_VALUE),
```
Comment (luckychess) [inline src/main/kotlin/sidechain/eth/util/DeployHelper.kt]: Do we really need so much tokens?

File: deploy/db/sql/init/populate.sql (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply deploy/db/sql/init/populate.sql]: Good point, I'll add task to remove PostgresWalletsProvider

File: src/main/kotlin/sidechain/eth/util/DeployHelper.kt (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/sidechain/eth/util/DeployHelper.kt]: Why not? For example, I want to deploy long term tokens in Ropsten for demo. 1000 is not enough definitely.

**Pull request merged**